src/client/components/atoms/collapse/expandMenu/style.scss

```diff
@@ -0,0 +1,6 @@
+.expand {
```

Comment:
in order to make the "Expandable" component reusable, I would not include this rules to the atom directly.

If needed, then please first:

1. Try to add those rules in parent
2. if not, then pass a new `className` prop, and give styles from parent component's style module

Reply:
In this case I would need help, I have tried to add those rules in a div container , and the rules doesn't work , we can solve this together?